utils/_context/_scenarios/__init__.py

```diff
@@ -870,6 +870,20 @@ class _Scenarios:
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
+    appsec_ato_sdk = EndToEndScenario(
```

Comment:
Can you add this scenario in `.github/workflows/run-end-to-end.yml` ?

Reply:
No, it's just a limitation in github action : it's not possible to do a matrix of step. So we can't get the scenarios list from a group, and dynamically create the list of step to run. We had to hard code a step for every scenario 😢